# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/neofetch/proc-fb-gpu.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- neofetch-7.1.0/neofetch~	2020-08-02 20:37:41.000000000 +0200
+++ neofetch-7.1.0/neofetch	2024-01-07 16:46:36.927215262 +0100
@@ -2385,6 +2385,7 @@
                                           '/"Display|"3D|"VGA/ {a[$0] = $1 " " $3 " " $4}
                                            END {for(i in a) {if(!seen[a[i]]++) print a[i]}}')"
             IFS=$'\n' read -d "" -ra gpus <<< "$gpu_cmd"
+            [ ! "$gpu" ] && IFS=$'\n' read -d "" -ra gpus <<< $(sed 's/^[[:digit:]] //' < /proc/fb)
 
             # Remove duplicate Intel Graphics outputs.
             # This fixes cases where the outputs are both
